pages/index.ejs

```diff
@@ -1,11 +1,45 @@
 ---
-title: Homepage
-description: Get your webiste up and running online in no time using current starter
-header: <fix>CMintS</fix> starter
+title: Privacy Manager
+description: Privacy Manager website
 ---
 
-<h2>{homepage[Page heading] Homepage}</h2>
-
-<p>{homepage-p[Paragraph in "Homepage" section] This is the homepage that is using <fix>EJS</fix>.}</p>
-
-<a href="/page2">{homepage-p-link[Link to the second page in "Homepage" section] Go to Page2}</a>
+<section class="col2 container">
+  <div>
+    <h2>{privacy-manager-header[Header of the section] Privacy Manager}</h2>
+    <ul>
+      <li>{privacy-manager-li1[Privacy Manager section bullet point] Privacy Management}</li>
+      <li>{privacy-manager-li2[Privacy Manager section bullet point] <fix>Cookie</fix> Management}</li>
+      <li>{privacy-manager-li3[Privacy Manager section bullet point] Browsing data deletion on browser start}</li>
+      <li>{privacy-manager-li4[Privacy Manager section bullet point] Network monitoring}</li>
+    </ul>
+    <a id="install-button" href="https://chrome.google.com/webstore/detail/privacy-manager/giccehglhacakcfemddmfhdkahamfcmd" target=_blank>Get Privacy Manager for Chrome</a>
+  </div>
+  <iframe class="tablet-hide end" id="promo-video" src="https://www.youtube.com/embed/iaKweivaOhI" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</section>
+<section class="bg-secondary">
+  <div class="col3 center container cards">
+    <h3 class="cell1-3">{what-you-get[Privacy Manager section h3 title] What <em>You Get</em>}</h3>
+    <div>
+      <img src="/images/trancparency.svg" alt="{what-u-get-trasparency-alt[Privacy manager transparency image alt text] Magnifier and newspaper}" />
+      <h4>{what-u-get-trasparency-heading[Privacy Manager section h4 title] Transparency}</h4>
```

Comment:
Suggestion. here and other similar places description can be like: "What you get" section h4 title

Reply:
Try using simlar pattern as in current suggestion -> https://github.com/Privacy-Managers/privacy-manager-website/pull/6#discussion_r494186301

I wonder why don't you think about: `"What You Get" section item heading` or `"What You Get" section item title`